### PR TITLE
feature/MC-12440-Create cluster for GCP API documentation

### PR DESCRIPTION
**Pull request by dpmurray21**
Added API docs for GCP create k8 cluster.

#### Changes made

- Added documentation for gcp create cluster.

#### Related PRs
- feature/MC-12440-create-kubernetes-cluster
https://github.com/cloudops/cloudmc-gcp-plugin/pull/346

### CODE REVIEW

File: source/includes/gcp/_clusters.md [hunk 1]
```diff
@@ -88,3 +88,52 @@ Attributes | &nbsp;
...
+
+Create a new cluster.
+
+Required | &nbsp;
+------- | -----------
+`name`<br/>*string* | The display name of the cluster.
+`shortRegion`<br/>*string* | A short version of the region name.
+`shortZone`<br/>*string* | A short version of the zone name.
+`initialClusterVersion`<br/>*string* | The version of GKE used for this cluster.
+`initialNodeCount` <br/>*string* | The number of nodes in the primary node pool of this cluster.
```
Comment (fyrnaga) [inline source/includes/gcp/_clusters.md]: Should mention that it needs to be greater than 0

File: source/includes/gcp/_clusters.md [hunk 2]
```diff
@@ -88,3 +88,52 @@ Attributes | &nbsp;
...
+`nodeType`<br/>*string* | The machine types of the nodes in the default node pool of this cluster.
+`network`<br/>*string* | The network that the Kubernetes cluster is in.
+`subnetwork`<br/>*string* | Subnetwork to which the cluster will belong.
```
Comment (fyrnaga) [inline source/includes/gcp/_clusters.md]: Maybe mention it must be part of the network.

**Review by GuillaumeVialle — APPROVED** [review]
LGTM

File: source/includes/gcp/_clusters.md (hunk 2 above)
Comment (dpmurray21) [reply source/includes/gcp/_clusters.md]: Added! c78a166019df93912fe2c09160b458c10127179b

File: source/includes/gcp/_clusters.md (hunk 1 above)
Comment (dpmurray21) [reply source/includes/gcp/_clusters.md]: Done! c78a166019df93912fe2c09160b458c10127179b

**Review by fyrnaga — APPROVED** [review]

**Pull request merged**